__tests__/app.test.js

```diff
@@ -274,4 +274,24 @@ describe('DELETE /api/comments/:comment_id', () => {
         expect(response.body.msg).toBe('Comment not found');
       });
   });
+});
+
+describe('GET /api/users', () => {
+  test('responds with an array of user objects', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(Array.isArray(users)).toBe(true);
+        users.forEach((user) => {
```

Comment:
when using a .forEAch in your testing, please make sure to check the length beforehand ! This is because if the array is empty, then the forEach auto-passes! Not ideal! :D